### DISCUSSION TITLE
Howto install foobar2000 to Monterey?

### DISCUSSION THREAD

**Discussion by offworldOwl** [The-Wineskin-Project/WineskinServer#249] [Q&A] [answered]
If anybody has found the right settings and got it working, I'd appreciate the info!

**Comment by emendelson** [comment, marked as answer]
I created a blank wrapper, started it, and told it to run foobar2000_v1.6.10.exe as the setup executable. For some reason this created a foobar2000 folder on my macOS desktop, not inside the wrapper. I used Wineskin's Install Software dialog to copy the folder into the wrapper (I chose the Copy Folder option), and Wineskin copied the folder to C:\Program Files. It now works perfectly. 

There's also a Mac version of the foobar2000 download page, but I haven't tried it, and I know you didn't ask about it.
